projects/mobidata-bw.json

```diff
@@ -0,0 +1,12 @@
+{
+  "name": "MobiData BW",
+  "desc": "MobiData BW - Daten und Dienste rund um Mobilität in Baden-Württemberg",
+  "url": "https://www.mobidata-bw.de/",
+  "licenses": [
+      "DL-DE->BY-2.0"
```

Comment:
I'm curious: Are there [SPDX license identifiers](https://spdx.org/licenses/) for *Datenlizenz Deutschland* licenses?

Reply:
I checked license list 3.12. There seems to be none.

Reply:
I have no explicit license list yet